cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/web/model/SortType.java

```diff
@@ -0,0 +1,5 @@
+package edu.asu.diging.cord19.explorer.web.model;
+
+public enum SortType {
+    publishYear, journal
```

Comment:
please follow our style guide

Reply:
I checked the enums in Monitor and they are defined in capital letters.